src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -33,10 +35,12 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
 
         return true;
     }
+    
+    String smiteReason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
 
     public static void smite(final Player player)
     {
-        TFM_Util.bcastMsg(player.getName() + " has been a naughty, naughty boy.", ChatColor.RED);
+        TFM_Util.bcastMsg(player.getName() + " has been a naughty, naughty boy.\nThey have thus been smitten/nReason - " + smiteReason, ChatColor.RED);
```

Comment:
@Scuph Broadcasts shouldn't have newlines. Additionally "They have thus been smitten" is unnecessary. If there's a reason present, it should be on the same line.